qemu/tests/netkvm_cpu_mapping.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_net
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Get cpu mapping info from traceview session, and check
+    whether it matches the vectors count.
+
+    :params test: the test object
+    :params params: the test params
+    :params env: test environment
+
+    """
+
+    # boot the vm with the  queues
+    queues = int(params["queues"])
+    error_context.context("Boot the guest with queues = %s" % queues,
```

Comment:
Since the 'queues' used 'int' type, please replace `%s` with `%d`

Reply:
This's does not matter. What she does is put format queues as string.